app/assets/stylesheets/components/_accordion.scss

```diff
@@ -16,6 +16,10 @@
 }
 
 .accordion {
+  border: 1px solid $border-color;
+  border-radius: $border-radius-md;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, .5);
```

Comment:
Looking at the latest style guide I think we can go without the `box-shadow` for now, otherwise lgtm!